Pear/Controller/HomeViewController.swift

```diff
@@ -317,20 +317,21 @@ extension HomeViewController: UNUserNotificationCenterDelegate {
                 // get rid of previously scheduled notifications
                 center.removeAllDeliveredNotifications()
                 center.removeAllPendingNotificationRequests()
-                self.scheduleNotifications(center: center, day: 2, hour: 8, title: "Meet your new pear!", body: "Set up this week's chat today 😊")
-                self.scheduleNotifications(center: center, day: 4, hour: 14, title: "Did you reach out yet?", body: "Choose a meeting time with your Pear before it's too late!")
-                self.scheduleNotifications(center: center, day: 6, hour: 12, title: "How's it going?", body: "New pairings will come out next week! ⌚️")
+                self.scheduleNotifications(center: center, day: 2, hour: 8, title: "Meet your new pear!", body: "Set up this week's chat today 😊", aboutPear: true)
+                self.scheduleNotifications(center: center, day: 4, hour: 14, title: "Did you reach out yet?", body: "Choose a meeting time with your Pear before it's too late!", aboutPear: true)
+                self.scheduleNotifications(center: center, day: 6, hour: 12, title: "How's it going?", body: "New pairings will come out next week! ⌚️", aboutPear: true)
+                self.scheduleNotifications(center: center, day: 0, hour: 0, title: "Are you running the latest version of Pear?", body: "Open TestFlight to check for new updates", aboutPear: false)
             }
         }
     }
 
-    private func scheduleNotifications(center: UNUserNotificationCenter, day: Int, hour: Int, title: String, body: String) {
+    private func scheduleNotifications(center: UNUserNotificationCenter, day: Int, hour: Int, title: String, body: String, aboutPear: Bool) {
```

Comment:
```suggestion
    private func scheduleNotifications(_ center: UNUserNotificationCenter, day: Int, hour: Int, title: String, body: String, aboutPear: Bool) {
```
so you can remove `center: center`

Reply:
kind of a nitpick but when `aboutPear` is false, the value of `day` and `hour` isn't used- maybe break it up into 2 functions `scheduleNotificationAboutPear` and `scheduleNotification(at time: ...)` so that's not the case. It would also make the function call itself shorter. 
__
up to you though, I dunno if that'll actually improve readability